ffv1.md

```diff
@@ -26,6 +26,12 @@ The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "S
 ## Definitions
 
 -------- --------------------------------------------------------------
+`Container`: Format whose specification describes how packets and metadata coexist in a file.
```

Comment:
Is the remaining `packets` here intentional?

Reply:
I guess typo, will change (frame and configuration record?)

Reply:
I updated this locally, but am also rewording as `Format whose specification describes how` is not true when Frames and Configuration Record are used with their FFV1 meansing; for instance ISOBMFF is a Container but its specification does not describe how to container the Configuration Record. Also 'file' is too specific as a Container could encapsulate FFV1 into a stream. Updating to:
> `Container`: Format that encapsulates `Frames` and (when required) a `Configuration Record` into a bitstream.